tests/python/test_resource_descriptor.py

```diff
@@ -0,0 +1,59 @@
+'''
```

Comment:
Can this go in a `tests/` directory in `/python`? So all the Python code is contained in the `python` dir, similar to how the Go tests are in the `go` dir with the corresponding packages.

Reply:
Yep, this sounds good.

Reply:
Not related to this PR, but should we also move the `examples/go` dir into the `go/` directory at some point? I've been thinking (in general) about where to house any existing and future examples for in-toto attestation use cases.